server/src/server.js

```diff
@@ -23,6 +23,14 @@ const jspath = "/js/",
   footer = "</body></html>",
   script = `
 function gotData(Handler, data) {
+  data.linkify = function (url) {
+    if (url.indexOf("?") !== -1) {
+      url += "&gitRevision=" + gitRevision;
+    } else {
+      url += "?gitRevision=" + gitRevision;
+    }
+    return url;
+  }
```

Comment:
oh, I missed that context from the diff.  That seems reasonable to me to have two copies then.